tests/integration/schema_registry/test_api.py

```diff
@@ -0,0 +1,257 @@
+from uuid import uuid1
```

Comment:
missing copyright header, shebang, doc string explaining the purpose of the file